gla_blockquote.module

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * Blockquote paragraphs component.
+ */
+
+use Drupal\Core\Entity\EntityTypeInterface;
+
+/**
+ * Implements hook_entity_bundle_field_info_alter().
+ */
+function gla_blockquote_entity_bundle_field_info_alter(&$fields, EntityTypeInterface $entity_type, $bundle) {
+  if ($entity_type->id() == 'paragraph' && $bundle === 'blockquote') {
```

Comment:
Could we have some extra comments here to describe why we are adding constraint to the Blockquote paragraph bundle?

Reply:
Perhaps mention that it uses the NoQuote constraint to ensures the string doesn't already contain the quote marks.

Reply:
Done.